plugins/rdap/help.md

```diff
@@ -663,6 +429,7 @@ _This plugin does not contain any troubleshooting information._
 
 # Version History
 
+* 2.0.0 - `Domain Lookup`, `IP Address Lookup`, `ASN Lookup`: Updated action outputs, refactored and prettified
```

Comment:
Reword 'Updated action outputs, refactored and prettified' -> 'Updated action outputs and refactored.